onnxscript/onnx_opset/__init__.py

```diff
@@ -183,7 +183,7 @@
         1,
     ): opset_ai_onnx_preview_training1,
 }
-default_opset: Opset14 = all_opsets[
+default_opset: Opset18 = all_opsets[
```

Comment:
Do we need this type-annotation? I suppose intellisense depends on this. On the other hand, this seems incorrect, since the type really depends on the installed ONNX version.

Maybe we don't even need this variable?

Reply:
Let me try that